src/stories/Blocks/autosuggest/Autosuggest.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  AutosuggestMaterial,
+  AutosuggestMaterialProps,
+} from "../../Library/autosuggest-material/AutosuggestMaterial";
+import {
+  AutosuggestText,
+  AutosuggestTextProps,
+} from "../../Library/autosuggest-text/AutosuggestText";
+
+export type AutosuggestProps = {
+  textProps: AutosuggestTextProps;
+  materialProps: AutosuggestMaterialProps["materialSuggestions"];
+};
+
+export const Autosuggest: React.FC<AutosuggestProps> = ({
+  textProps,
+  materialProps,
+}) => {
+  return (
+    <ul className="autosuggest pb-16">
+      <AutosuggestText textSuggestions={textProps.textSuggestions} />
+      <AutosuggestMaterial materialSuggestions={materialProps} />
+      <li className="autosuggest__divider" />
```

Comment:
I think it makes sense. 
I have added the dividers using just styling. It introduces a problem of its own that I didn't dive deeper into because it's a bit of a black hole. But the problem is that we always render the divider underneath material suggestions if there are any text suggestions in the top of the autosuggest - assuming that if there are any, there will also be material type filtered suggestions. 
However, in the unlikely case there are no material type filtered suggestions and there are text suggestions present - there will still be a divider rendered underneath the material suggestions as so:
![image](https://user-images.githubusercontent.com/28546954/226669601-56b8dc22-ee0b-4269-9384-6524ca2c2dc3.png)